documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -1391,6 +1391,22 @@ Only alphanumeric characters and underscores should be used.
 |
 |An optional comma-separated list of regular expressions that match fully-qualified table identifiers for tables to be excluded from monitoring; any table not included in the blacklist will be monitored. Each identifier is of the form _schemaName_._tableName_. May not be used with `table.whitelist`.
 
+|`column.mask.hash._hashAlgorithm_.with.salt._salt_`
+|_n/a_
+|An optional comma-separated list of regular expressions that match the fully-qualified names of character-based columns whose values should be pseudonyms in the change event message values with a field value consisting of the hashed value using the algorithm `_hashAlgorithm_` and salt `_salt_`.
+Based on the used hash function referential integrity is kept while data is pseudonymized. Supported hash functions are described in the {link-java7-standard-names}[MessageDigest section] of the Java Cryptography Architecture Standard Algorithm Name Documentation.
+The hash is automatically shortened to the length of the column.
+
+Multiple properties with different lengths can be used in a single configuration, although in each the length must be a positive integer or zero. Fully-qualified names for columns are of the form _databaseName_._tableName_._columnName_, or _databaseName_._schemaName_._tableName_._columnName_.
+
+Example:
+
+    column.mask.hash.SHA-256.with.salt.CzQMA0cB5K = databaseName.tableOrder.customerName, databaseName.tableShipment.customerName
```

Comment:
Thanks for making the salt explicit, that was the piece I was missing before. In the tests you also use the special value "randomSalt", this should be documented here, too? Also I'd suggest `inventory.orders.customerName` as the value (i.e. use actual names, and also remove the "table" prefix which you typically wouldn't have in your table names.

Reply:
I also changed the test because there is no special value "randomSalt". It was just a randomly selected text.

Reply:
Doh, oook. This was *really* confusing, LOL :)